packages/celotool/src/lib/oracle.ts

```diff
@@ -47,40 +42,24 @@ interface OracleConfig {
 /**
  * Env vars corresponding to each value for the AzureClusterConfig for a particular context
  */
-const oracleAzureContextClusterConfigEnvVars: {
-  [key in keyof typeof OracleAzureContext]: { [k in keyof AzureClusterConfig]: string }
-} = {
-  [OracleAzureContext.PRIMARY]: {
-    subscriptionId: envVar.ORACLE_PRIMARY_AZURE_SUBSCRIPTION_ID,
-    tenantId: envVar.ORACLE_PRIMARY_AZURE_TENANT_ID,
-    resourceGroup: envVar.ORACLE_PRIMARY_AZURE_KUBERNETES_RESOURCE_GROUP,
-    clusterName: envVar.ORACLE_PRIMARY_AZURE_KUBERNETES_CLUSTER_NAME,
-  },
-  [OracleAzureContext.SECONDARY]: {
-    subscriptionId: envVar.ORACLE_SECONDARY_AZURE_SUBSCRIPTION_ID,
-    tenantId: envVar.ORACLE_SECONDARY_AZURE_TENANT_ID,
-    resourceGroup: envVar.ORACLE_SECONDARY_AZURE_KUBERNETES_RESOURCE_GROUP,
-    clusterName: envVar.ORACLE_SECONDARY_AZURE_KUBERNETES_CLUSTER_NAME,
-  },
+const oracleContextAzureClusterConfigDynamicEnvVars: { [k in keyof AzureClusterConfig]: DynamicEnvVar } = {
+  subscriptionId: DynamicEnvVar.ORACLE_AZURE_SUBSCRIPTION_ID,
+  tenantId: DynamicEnvVar.ORACLE_AZURE_TENANT_ID,
+  resourceGroup: DynamicEnvVar.ORACLE_AZURE_KUBERNETES_RESOURCE_GROUP,
+  clusterName: DynamicEnvVar.ORACLE_KUBERNETES_CLUSTER_NAME,
+}
+
+interface OracleIdentityConfig {
```

Comment:
Given that only one of these will be defined, perhaps it's cleaner to define two separate objects.
```
interface KeyVaultIdentityConfig {
  addressAzureKeyVaults: string
}
interface MnemonicIdentityConfig {
  addressesFromMnemonicCount: string
}
type OracleIdentityConfig = KeyVaultIdentityConfig | MnemonicIdentityConfig
```